### PR TITLE
conan_info review

### PR DESCRIPTION
**Pull request by memsharded**

### CODE REVIEW

File: howtos/define_abi_compatibility.rst [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+
+
+Now the computed package ID is different, that means that we need a different binary package for GCC 4.4
+
+The same way we have adjusted the ``self.info.settings`` we could set the ``self.info.options`` values if necessary.
+
+
+.. _controlling_requires_compatibility:
+
+The problem of dependencies in binaries
```
Comment (lasote) [inline howtos/define_abi_compatibility.rst]: It sounds weird

File: howtos/define_abi_compatibility.rst [hunk 2]
```diff
@@ -0,0 +1,386 @@
...
+
+It could happen that the API exposed by ``MyOtherLib``in public headers has changed, but without
+affecting the ``MyLib/1.0`` binary, for whatever reasons, like changes consisting on new functions,
+not used by MyLib. The same reasoning would still be valid if ``MyOtherLib`` was header only.
+
+But what if one header file of ``MyOtherLib``, named ``myadd.h`` has changed from ``2.0``:
+
+ .. code-block:: cpp
+ 
+    int addition (int a, int b) { return a - b; }
```
Comment (lasote) [inline howtos/define_abi_compatibility.rst]: At least use a template to justify the implementation in a header.

Comment (memsharded) [inline howtos/define_abi_compatibility.rst]: Automatic inlining is more than a reason to keep simple function implementation in headers

Comment (lasote) [inline howtos/define_abi_compatibility.rst]: Whatever. C++: the unlimited source of excuses to keep dirtying your source code XD

**Pull request merged**